src/pages/STG_PanelSchedule.page

```diff
@@ -49,7 +49,19 @@
                 </div>
             </div>
 
-            <div class="slds-grid slds-gutters_small">            
+            <div class="slds-grid slds-gutters_small">
+                <div class="slds-col slds-has-flexi-truncate slds-grid" role="listitem">
+                    <div class="slds-form-element">
+                        <apex:outputLabel value="{!$ObjectType.Customizable_Rollup_Settings__c.Fields.Rollups_AcctContactSoftCredit_Batch_Size__c.Label}" for="tbxAccContactSoft" styleClass="slds-form-element__label" />
+                        <div class="slds-form-element__control">
+                            <apex:outputField value="{!stgService.stgCRLP.Rollups_AcctContactSoftCredit_Batch_Size__c}" rendered="{!isReadOnlyMode}" styleClass="slds-form-element__static" />
+                            <apex:inputField value="{!stgService.stgCRLP.Rollups_AcctContactSoftCredit_Batch_Size__c}" rendered="{!isEditMode}" id="tbxAccContactSoft" styleClass="slds-input" />
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <div class="slds-grid slds-gutters_small">
```

Comment:
@sam-knox Let me know if I positioned it in the correct spot.  We can move it up or down